### PR TITLE
Update flask-base and ubuntu lts to jammy

### PR DESCRIPTION
**Pull request by edlerd**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement
 
 # Build stage: Install yarn dependencies
 # ===
-FROM node:19 AS yarn-dependencies
+FROM node:18 AS yarn-dependencies
```
Comment (carkod) [inline Dockerfile:14]: Does it not work with 19?

**Review by carkod — APPROVED** [review]
LGTM

**Comment by carkod** [comment]
Could you also update Vanilla? so we don't have [two conflicting PRs](https://github.com/canonical/anbox-cloud.io/pull/281/files)

File: Dockerfile (hunk 1 above)
Comment (edlerd) [reply Dockerfile:14]: I learned from vanilla, that we tend to use only LTS versions, hence I used 18 here.

**Comment by edlerd** [comment]
> Could you also update Vanilla? so we don't have [two conflicting PRs](https://github.com/canonical/anbox-cloud.io/pull/281/files)

I merged both after another. Dependabot is surprisingly fast in adjusting after one PR is merged.

**Pull request merged**